openshift/openshift-ansible-contrib
# Specify different image names for roles

#### What does this PR do?
User can now use different images for different roles (servers).

#### How should this be manually tested?
1) Print out available Glance images ( `openstack image list`).
2) In `<path to inventory>/group_vars/all.yml`, uncomment line(s) with variables for specifying image names (`openstack_<role name>_image_name`) and set the value to one of the other available images.
3) Run the provisioning.
4) Display servers with `openstack server list` and notice that specified roles have changed image names to those set in `all.yml`. The rest of the servers use the default image (defined in `openstack_default_image_name`).

#### Is there a relevant Issue open for this?
--

#### Who would you like to review this?
cc: @tomassedovic @bogdando PTAL
